src/style.css

```diff
@@ -0,0 +1,45 @@
+/* reset css */
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
```

Comment:
Do not use such rules as you did for `* {}` selector
Only `box-sizing` is allowed